packages/nocodb/src/db/field-handler/handlers/formula/formula.general.handler.ts

```
@@ -0,0 +1,56 @@
+          column.uidt = UITypes.DateTime;
+          return options.fieldHandler.verifyFilter(filter, column, options);
+        }
+        case FormulaDataTypes.INTERVAL: {
+          column.uidt = UITypes.Time;
+          return options.fieldHandler.verifyFilter(filter, column, options);
+        }
+        case FormulaDataTypes.NUMERIC: {
+          column.uidt = UITypes.Decimal;
+          return options.fieldHandler.verifyFilter(filter, column, options);
+        }
+        case FormulaDataTypes.STRING:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)